src/ui/services/home.ui-service.ts

```diff
@@ -0,0 +1,22 @@
+import { Page } from "@playwright/test";
+import { ModuleName } from "types/home.types";
+import { CustomersPage } from "ui/pages/customers/customers.page";
+import { HomePage } from "ui/pages/home.page";
+
+export class HomeUIService {
+  homePage: HomePage;
+  customersPage: CustomersPage;
+  constructor(private page: Page) {
+    this.customersPage = new CustomersPage(page);
+    this.homePage = new HomePage(page);
+  }
+
+  async openModule(moduleName: ModuleName) {
+    await this.homePage.clickModuleButton(moduleName);
+    switch (moduleName) {
+      case "Customers":
+        await this.customersPage.waitForOpened();
+        break;
+    }
+  }
```

Comment:
Как вариант без switch конструкции

```suggestion
  async openModule(moduleName: ModuleName) {
    await this.homePage.clickModuleButton(moduleName);
    await this[`${moduleName.toLowerCase()}Page`].waitForOpened();
  }
```